csunplugged/classic/search_indexes.py

```diff
@@ -0,0 +1,31 @@
+"""Search index for ClassicPage model."""
+
+from haystack import indexes
+from classic.models import ClassicPage
+
+
+class ClassicPageIndex(indexes.SearchIndex, indexes.Indexable):
+    """Search index for ClassicPage model."""
+
+    text = indexes.CharField(document=True, use_template=True)
+
+    def prepare(self, obj):
+        """Set boost of ClassicPage model for index.
+
+        Args:
+            obj (ClassicPage): ClassicPage object.
+
+        Returns:
+            Dictionary of data.
+        """
+        data = super(ClassicPageIndex, self).prepare(obj)
+        data["_boost"] = 0.6
```

Comment:
how did you calculate this number?

Reply:
alternatively, how much of a boost is this compared to the others? will this bump classic results to the top of the list (and is that what we are hoping for?)

Reply:
I picked numbers based off what I thought things should be ranked (this is the base ranking, then text matches will change the score accordingly), bigger number is better:

- Topic = 2
- Unit plan = 1.5
- Lesson = 1.2
- Resources = 1 (default)
- General pages = 0.8
- Curriculum integration = 0.7
- Classic pages = 0.6
- Programming challenge = 0.4

Pages seem to be coming out in a useful order.

Reply:
that makes sense to me :+1: